src/main/kotlin/com/procurify/flagcounter/FlagCounter.kt

```diff
@@ -88,12 +88,16 @@ class FlagCounter(
      * All flags should have the same owner
      */
     private fun formatTeamMessage(teamFlags: List<FlagDetail>): String {
-        val removeCount = teamFlags.filter { it.status == Status.REMOVABLE }.size
+        val removeCount = removableFlagCount(teamFlags) 
         // TODO Parameterize link url based on project/environment configuration
         return """Hey ${teamFlags.firstOrNull()?.owner?.name ?: "team"}!
            |Launch Darkly thinks $removeCount of your ${teamFlags.size} flags could be ready for removal.
            |Take a look ${flagReader.flagListUrl}""".trimMargin()
```

Comment:
Does Kotlin allow 
```
Launch Darkly thinks $removableFlagCount(teamFlags)
```
?  If it does, can just condense this and remove `removeCount`.

Reply:
yes, you will have to wrap it in braces like `${teamFlags.size}`